BUIP-UAHF/buip-uahf-technical-spec.md

```diff
@@ -151,6 +153,9 @@ NOTE 2: The client shall still accept transactions whose signatures
 verify according to pre-fork rules, subject to the additional OP_RETURN
 constraint introduced by REQ-6-1.
 
+NOTE 3: If bit 6 is not set, only the traditional non-shifted nHashType
```

Comment:
We need to correct the wording here, it was not meant in the bitshifting sense, but in the 'displace from current hash' meaning.

We will be effectively adding a value (or bitwise ORing) in the code.